packages/core/tsconfig.json

```diff
@@ -0,0 +1,103 @@
+{
+  "include": ["src"],
```

Comment:
Can we extend the root `tsconfig.json` from `next` branch after rebasing if it makes sense?